src/qibotn/backends/quimb.py

```diff
@@ -43,21 +48,23 @@ def configure_tn_simulation(
         self.max_bond_dimension = max_bond_dimension
         self.n_most_frequent_states = n_most_frequent_states
 
-    def setup_backend_specifics(self, qimb_backend="numpy"):
+    def setup_backend_specifics(self, qimb_backend="numpy", optimizer="auto-hq"):
```

Comment:
```suggestion
    def setup_backend_specifics(self, qimb_backend="numpy", contractions_optimizer="auto-hq"):
```

Reply:
If this argument is impacting on the overall performances of the contraction operations, I am fine in adding it. 
I would be more explicit in the name (see suggestion).